Base/usr/share/man/man1/su.md

```diff
@@ -1,41 +1,42 @@
 ## Name
 
-su - run a command with substitute user
+su - switch to another user
 
 ## Synopsis
 
-```
-$ su [user]
+```**sh
+$ su [--help] [--version] <user>
 ```
 
 ## Description
 
-**su** allows commands to be run with a substitute user.
-
-When called with no user-specified, **su** defaults to running an interactive shell as *root*. 
+`su` switch to another user.
 
-## Options
+When called with no user-specified, `su` defaults to switch to the *root* user. 
+**NOTE**
 
-- **--help**
+* Need to enter the password if the user switch to has one.
 
-  Display help message and exit.
+## Options
 
-- **--version**
+* `--help`: Display help message and exit.
 
-  Display version information and exit.
+* `--version`: Display version information and exit.
 
 ## Arguments
 
-- user: User to switch to (defaults to user with UID 0)
+* `user`: User to switch to (defaults to the user with UID 0)
 
 ## Examples
 
-```
-# Switch to root user
+Switch to root user
+
+```sh
 $ su
+```
 
-# Switch to another user
-$ su nona
+Switch to another user
 
+```sh
+$ su nona
 ```
```

Comment:
I think these were fine as-is.

Reply:
No, all examples should be wrapped in code blocks if appropriate. This is correct. Again -> see existing man pages and their style.